### DISCUSSION TITLE
Doubts

### DISCUSSION THREAD

**Discussion by sethdivyansh** [shivansh-bhatnagar18/multiplayer-uno#141] [General]

**Comment by Vermouth10** [comment]
Hi all developers, I am an IT student and I consider to develop a special UNO game which cover the element of a Chinese Virtual Idol team which named "Asoul" with my friends, and then I found your UNO game project here and I think the technique stack is appropriate. I'd like to ask whether your project is open source? May I fork your project as the bedrock to develop and optimise so that achieve our requirement? Cause after development, our game will be provided to Asoul fans to play together for free(no charge), thus I post this enquiry to acquire the authorization and avoid copyright infringement. Thanks for your reading.
